java/ql/test/TestUtilities/InlineExpectationsTest.qll

```diff
@@ -202,6 +202,7 @@ private predicate getAnExpectation(
       column = TDefaultColumn() and
       exists(int end |
         end = getEndOfColumnPosition(0, content) and
+        not exists(string name | exists(TNamedColumn(name)) and content.indexOf(name + ":") < end) and
```

Comment:
Remove this commit. (You can add a space between `$` and `MISSING` to make it work)